apps/src/code-studio/components/Libraries/LibraryManagerDialog.jsx

```diff
@@ -12,19 +51,25 @@ export default class LibraryManagerDialog extends React.Component {
 
   state = {
     importLibraryId: '',
-    libraries: []
+    libraries: [],
+    classLibraries: []
   };
 
   componentDidMount = () => {
+    let libraryClient = new LibraryClientApi();
     this.setState({libraries: dashboard.project.getProjectLibraries() || []});
+    this.setState({classLibraries: libraryClient.getClassLibraries() || []});
   };
 
   setLibraryToImport = event => {
     this.setState({importLibraryId: event.target.value});
   };
 
-  addLibrary = () => {
-    let libraryClient = new LibraryClientApi(this.state.importLibraryId);
+  addLibrary = event => {
+    let libraryToImport = event.target.value
+      ? event.target.value
+      : this.state.importLibraryId;
```

Comment:
When would this be used instead of `event.target.value`?

Reply:
importLibraryId is the value of the input box at the bottom of the screenshot (above). Also, it turns out event.target.value actually won't be used. I have a follow-up PR that passes Library.channel for each libraryListItem.